### ISSUE TITLE
lsinfo results are not the same as mpd

### ISSUE THREAD

**Issue by gilphilbert** [cotko/mpd-api#5]
When I telnet directly to the mpd server and issue lsinfo for a given URI, I get back a response that includes the metadata about the song at that URI:
```
OK MPD 0.22.4
lsinfo "music-lossless/Chris De Burgh/Spanish Train and Other Stories/10. Just Another Poor Boy - Chris de Burgh.flac"
file: music-lossless/Chris De Burgh/Spanish Train and Other Stories/10. Just Another Poor Boy - Chris de Burgh.flac
Last-Modified: 2020-06-12T18:40:18Z
Format: 44100:16:2
Title: Just Another Poor Boy
Artist: Chris de Burgh
Album: Spanish Train And Other Stories
Genre: Pop
AlbumArtist: Chris De Burgh
Disc: 1
Date: 1975
Track: 10
Time: 282
duration: 282.066
OK
```

However, issuing an lsinfo command via mpd-api I get very different results:
```
{
  playlist: [],
  file: [
    'music-lossless/Chris De Burgh/Spanish Train and Other Stories/10. Just Another Poor Boy - Chris de Burgh.flac'
  ],
  directory: []
}
```
Using the underlying mpd2 instance, I can get the results expected (but now I must filter them myself)
```
tmp = await mpdc.sendCommand(mpd.cmd('lsinfo', fileList[i].file))
console.log(tmp)
```

```
music-lossless/Chris De Burgh/Spanish Train and Other Stories/10. Just Another Poor Boy - Chris de Burgh.flac
file: music-lossless/Chris De Burgh/Spanish Train and Other Stories/10. Just Another Poor Boy - Chris de Burgh.flac
Last-Modified: 2020-06-12T18:40:18Z
Format: 44100:16:2
Title: Just Another Poor Boy
Artist: Chris de Burgh
Album: Spanish Train And Other Stories
Genre: Pop
AlbumArtist: Chris De Burgh
Disc: 1
Date: 1975
Track: 10
Time: 282
duration: 282.066
```
Is this deviation intentional? According to the docs it's supposed to be a direct mapping but the results are obviously very different.

**Comment by cotko** [comment]
@gilphilbert you're right, that response makes sense if you're not passing any URI to the lsinfo command (that response differs from responses when you pass URI).

I've fixed this issue, response now contains all meta info for all items (file/directory/playlist).
I've also added new method `db.songinfo('path to song')` which can be used if you want to get info for certain song only .

**Comment by gilphilbert** [comment]
Wow, that was fast! Thanks 👍🏼

**Comment by andersevenrud** [comment]
@cotko Looks like the `songinfo` is missing from type definitions :)

**Comment by cotko** [comment]
@andersevenrud you're right, just added and released v1.1.1 :)

**Comment by andersevenrud** [comment]
@cotko Awesome! Appreciate the quick response and resolution :rocket:
